base/iddict.jl

```diff
@@ -6,7 +6,23 @@
 `IdDict{K,V}()` constructs a hash table using object-id as hash and
 `===` as equality with keys of type `K` and values of type `V`.
 
-See [`Dict`](@ref) for further help.
+See [`Dict`](@ref) for further help. In the example below, The `Dict`
+keys all hash to the same value as they are inserted and type-promoted,
```

Comment:
There isn't really any type promotion going on here though, the values are all `isequal` and hash the same.

Reply:
Thanks for the catch. Should be fixed.